modules/combined/examples/periodic_strain/global_strain_pfm.i

```diff
@@ -341,7 +322,7 @@
 []
 
 [UserObjects]
-  [./appl_stress]
+  [./global_strain_uo]
     type = GlobalStrainUserObject
     execute_on = 'Initial Timestep_begin Linear'
```

Comment:
Shouldn't this be `linear nonlinear`?

Reply:
Ideally, it should be calculated at nonlinear as well. But, for the test problems it does not affect the convergence/result.